cms/templates/js/add-xblock-component-menu-problem.underscore

```diff
@@ -4,9 +4,6 @@
             <a class="link-tab" href="#tab1"><%= gettext("Common Problem Types") %></a>
         </li>
         <li>
-            <a class="link-tab" href="#tab2"><%= gettext("Common Problems with Hints and Feedback") %></a>
```

Comment:
Ah I assumed you had code reviewed this. The template defines what the tabs are, and the yaml template declares what tab it wants to be in.